docs/pages/app.md

```diff
@@ -87,3 +93,28 @@ function handleURL(context) {
 Open the following link to display a _Hello World_ message within an open document. Provide any URL encoded string with the `msg` query parameter.
 
 [`sketch://plugin/com.example.sketch.messenger/message.show?msg=Hello%20World`](sketch://plugin/com.example.sketch.messenger/message.show?msg=Hello%20World)
+
+## Add a Remote Library <a name="addremotelibrary"></a>
+
+You can add a Remote Library to Sketch by providing a url to an appcast file.
+
+### Parameters
+
+Provide an `encoded appcast url` parameter to the `add-library` command.
```

Comment:
### `url` string

URL-encoded string specifying the address of the appcast RSS file.

Reply:
Changed.